solutions/src/test/java/ru/sberbank/school/task06/CollectionUtilsTest.java

```diff
@@ -0,0 +1,214 @@
+package ru.sberbank.school.task06;
+
+import lombok.Getter;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static ru.sberbank.school.task06.CollectionUtils.*;
+
+/**
```

Comment:
First sentence of Javadoc is missing an ending period.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-387830510>*